Directory.Build.props

```diff
@@ -481,7 +481,7 @@
     <!-- We ignore all documentation related warnings. -->
     <NoWarn>$(NoWarn);1573;1591</NoWarn>
   </PropertyGroup>
-
+  <Target Name="Test2" />
```

Comment:
Nice try to infiltrate test code into our msbuild logic 😉

Reply:
Woops thanks for catching this.  Was  to eager to get the PR going before leaving the house.